src/data/Pager/Pager.tsx

```diff
@@ -152,7 +152,7 @@ export const Pager: FC<PagerProps> = ({
       {startArrow && (
         <Button
           className={twMerge(theme.control, theme.firstPage)}
-          variant="text"
+          variant="ghost"
```

Comment:
will this cause problems when upgrading? if so, we'll need to document in migration notes